raiden_contracts/contract_manager.py

```diff
@@ -47,21 +49,26 @@ class ContractManagerVerificationError(RuntimeError):
 
 
 class ContractManager:
-    def __init__(self, path: Union[Path, Dict[str, Path]]) -> None:
+    def __init__(self, flavor: Flavor, path: Union[Path, Dict[str, Path]]) -> None:
```

Comment:
Github does not allow me to comment on the line itself since it's not touched by this PR but the `Flavor` Enum. Can you make the values of the enum all CAPS? So like `UNLIMITED = 1` e.t.c.? This is the convention we follow in the Raiden codebase

Reply:
Oh I see.  I'll follow that in this PR.